### DISCUSSION TITLE
Outreachy intern

### DISCUSSION THREAD

**Discussion by JesperKawuma** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#31] [Outreachy 2022] [answered]
Hello everyone, I am Jesper Kawuma from Uganda an Outreachy applicant ready to contribute to the **OSCSA** community **relentlessly**. I am also **open to learning** any skill, tool, or anything for the betterness of this community. Hope to work with you soon. Thanks

**Comment by BatoolMM** [comment, marked as answer]
Thank you @JesperKawuma for all the PRs you submitted so far!
